### DISCUSSION TITLE
Downloaded Replay Scores show in Local

### DISCUSSION THREAD

**Discussion by Mine181** [ppy/osu#18782] [Q&A] [answered]
Downloading a replay from another user adds it to your local list
![image](https://user-images.githubusercontent.com/83498665/174633703-443ee606-9861-451a-aa0b-e17d27558663.png)
Just asking to make sure that it's a bug and hasn't been submitted before.

**Comment by bdach** [comment, marked as answer]
that is intended behaviour.
